src/core/menu_items/RFMenu.cpp

```diff
@@ -15,7 +15,11 @@ void RFMenu::optionsMenu() {
     };
 
     delay(200);
-    loopOptions(options,false,true,"Radio Frequency");
+    String txt = "Radio Frequency";
+    if(RfModule)    txt+=" (CC1101)"; // Indicates if CC1101 is connected
```

Comment:
there's an Enum for the rfid and rf modules

Reply:
hmmmmmmmmm
I put it there to show the selected model on the SubMenu screen